.gitignore

```diff
@@ -2,6 +2,6 @@
 
 .xml
 
-.idea
+.idea/
 
-geckodriver.log
\ No newline at end of file
+*.log
```

Comment:
pep8: extra line needed at the end of file.

Reply:
Done